app/assess/models/question.py

```diff
@@ -16,7 +16,7 @@ def from_json(data: dict):
             for field_data in data["fields"]:
                 field = QuestionField.from_json(field_data)
                 question.add_field(field)
-
+        # print(question)
```

Comment:
remove

Reply:
Already spotted and removed

Reply:
Have you pushed this removal? Still visible here for some reason...?

Reply:
I just pushed the changes now, it takes ages due to the zap scan so wanted to push the changes altogether